manifests/example/upload-datavolume.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: upload-datavolume
+spec:
+  source:
+      upload:
+        shouldUpload: true
```

Comment:
Would there ever be a situation where this should be false?  Can't we just have an empty dict such as:

```
source:
    upload: {}
```

Reply:
done